<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTPlotter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenProject"/>
    <addaction name="menuRecentProjects"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionSaveProjectAs"/>
    <addaction name="separator"/>
    <addaction name="actionImportData"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <widget class="QMenu" name="menuRecentProjects">
     <property name="title">
      <string>Recent Projects</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionClearRecentProjects"/>
    </widget>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionPlotOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
   <addaction name="actionSaveProjectAs"/>
   <addaction name="separator"/>
   <addaction name="actionPause"/>
   <addaction name="separator"/>
   <addaction name="actionResetZoom"/>
   <addaction name="separator"/>
   <addaction name="actionImportData"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionConfigurePlots"/>
  </widget>
  <action name="actionOpenCSV">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/import.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>Open CSV</string>
   </property>
  </action>
  <action name="actionConfigurePlots">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/settings.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>Configure Plots</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/pause.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionResetZoom">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/refresh.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>Reset Zoom</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/export.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/open.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/save.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveProjectAs">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/save-as.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionImportData">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/import.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>Import data...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/exit.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPlotOptions">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/settings.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>Plot options</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/help.svg</normaloff>
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClearRecentProjects">
   <property name="text">
    <string>Clear List</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
